consent-ws/src/main/java/org/broadinstitute/consent/http/resources/NihAccountResource.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.consent.http.resources;
+
+import org.broadinstitute.consent.http.service.NihAuthApi;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+@Path("{api : (api/)?}nih-login/")
```

Comment:
Nit: You probably don't need this regex in the path. We used this technique for handling basic authentication and I think the pattern stuck. It can probably just be `@Path("api/nih-login/")`. This should be true in lots of other cases too.